diff -uNr urtconnector-0.5.1.orig//src/main_window.cpp urtconnector-0.5.1//src/main_window.cpp
--- urtconnector-0.5.1.orig//src/main_window.cpp	2010-09-07 01:33:59.000000000 -0500
+++ urtconnector-0.5.1//src/main_window.cpp	2010-11-08 19:42:12.435622729 -0500
@@ -369,7 +369,7 @@
     };
     
 #if defined(Q_OS_UNIX)
-    QString default_qstat = "/usr/bin/qstat";
+    QString default_qstat = "/usr/bin/qstat-quake";
 #elif defined(Q_OS_WIN)
     QString default_qstat = "qstat.exe";
 #elif defined(Q_OS_MAC)
